### DISCUSSION TITLE
"webdriverio" has no exported member named 'Matches'

### DISCUSSION THREAD

**Discussion by vincent-lyon** [webdriverio/webdriverio#12201] [Q&A] [answered]
Hi,
For a few days VSCode has been higlighting my import of the Matches type (used in mocks) with the error `"webdriverio" has no exported member named 'Matches'`.
I did not change anything, except updating the expect-webdriverio package from 4.2.7 to 4.6.0.

I'm not sure what caused this error to suddenly show up, but my guess is that it has more to do with VSCode than wdio, since there's no error when running tests. Or maybe it has to do with the tsconfig types ?

Here is a sample of how I use the Matches type
```typescript
import { Matches } from 'webdriverio';

const mock = await browser.mock(url, { method: 'GET });
mock.respond(buildMockResponse);

function buildMockResponse(request: Matches): void {
    /* do something with request like parse the url or parse the payload */
}
```

Here is my types config in the tsconfig.json file
```json
"types": [
  "node",
  "@wdio/cucumber-framework",
  "@wdio/globals/types",
  "@wdio/types",
  "expect-webdriverio",
  "wdio-wait-for"
],
```

And the package.json dev dependencies
```json
"devDependencies": {
    "@types/node": "14.10.2",
    "@wdio/cli": "8.15.4",
    "@wdio/cucumber-framework": "8.15.4",
    "@wdio/local-runner": "8.15.4",
    "@wdio/spec-reporter": "8.15.0",
    "cucumber-json-merge": "0.0.8",
    "expect-webdriverio": "4.6.0",
    "ts-node": "10.9.1",
    "typescript": "4.8.4",
    "wdio-cucumberjs-json-reporter": "5.1.6",
    "wdio-wait-for": "3.0.6"
  }
```

The error is resolved if I change the import with the full path, but it does not seem as a satisfying solution (though I might use it if I can't find anything else)
`import { Matches } from 'webdriverio/build/utils/interception/types';`

Has anyone encountered this and found a solution besides putting the full path ? Maybe I should update the other packages as well ?

**Comment by christian-bromann** [comment, marked as answer]
I just pushed a fix to allow importing the `Matches` type from the `webdriverio` package. Thanks for reporting.

**Reply by vincent-lyon** [reply]
The error disappeared with v8.31.1, thank you ! Marking as resolved
